recipes/mumemto/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "mumemto" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vikshiv/mumemto/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e0a42fd5d3afde6366dbc43bb16cc38b5a0c87468a4c6978c4225d361d789f95
+
+build:
+  number: 0
+  pkg_format: '.conda'
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
```

Comment:
```suggestion
    - {{ pin_subpackage(name, max_pin='x') }}
```

Reply:
As I understand it, changing to `'x'` would mean that this package would get pinned to the major version? I anticipate that "minor" (_.X._) versions will be substantial and affect usage, so I would rather pin to the second version number. Is this a correct understanding? Thank you.

Reply:
I apologize, I re-read the pull request information. My evidence for the minor version breakages is the current update. From v1.0.0 (not on bioconda) to v1.1.0 (first version on bioconda), the usage (flag parameters) and output format are significantly different and not interoperable. However, this is still an initial release so we want to keep the v1 tag. Thus our minor version updates will be substantial and break usage potentially, while our path updates will be for bug fixes.